app/ts/utils/augurContractUtils.ts

```diff
@@ -577,3 +596,37 @@ export const getMarketRepBondForNewMarket = async (client: ReadClient, universe:
 		args: []
 	})
 }
+
+export const getRepTotalTheoreticalSupply = async (client: ReadClient, repToken: AccountAddress) => {
+	return await client.readContract({
+		abi: REPUTATION_TOKEN_ABI,
+		functionName: 'getTotalTheoreticalSupply',
+		address: repToken,
+		args: []
+	})
+}
+
+export const getTotalSupply = async (client: ReadClient, repToken: AccountAddress) => {
+	return await client.readContract({
+		abi: REPUTATION_TOKEN_ABI,
+		functionName: 'totalSupply',
+		address: repToken,
+		args: []
+	})
+}
+
+export const getWinningChildUniverse = async (client: ReadClient, universe: AccountAddress) => {
+	try {
+		return await client.readContract({
+			abi: UNIVERSE_ABI,
+			functionName: 'getWinningChildUniverse',
+			address: universe,
+			args: []
+		})
+	} catch(error: unknown) {
+		if (error instanceof ContractFunctionExecutionError) { // fails if we don't know yet which universe won
+			return undefined
+		}
```

Comment:
Is there no way for us to do a read and check this first, rather than throwing an exception?